### PR TITLE
[Doc][Manifest] add no-affine variants for GroupNormFwdOp / InstanceNormFwdOp

### PR DESCRIPTION
**Pull request by lcy-seso**
Closes tile-ai/TileOPs#1267

## Summary

- Add `GroupNormFwdOpNoAffine` and `InstanceNormFwdOpNoAffine` variant entries to `tileops/manifest/normalization.yaml`, each linked via `variant_of` to the existing affine primary.
- Variants drop `weight` / `bias` from the signature (matching `torch.nn.GroupNorm(affine=False)` and `torch.nn.InstanceNorm*(affine=False)`), per project rule R16 ("No `Optional[Tensor]` — split into variants via `variant_of`").
- Variant `roofline` reflects the no-affine cost model (3 fp ops/elem vs 5; reads x + writes output, no weight/bias bytes); workloads mirror the primary's CV shapes.
- Primary entries (`GroupNormFwdOp`, `InstanceNormFwdOp`) are byte-stable — no edits to existing keys.
- Variants ship as `status: spec-only`; implementation does not yet honor the no-affine signature, so promotion to `implemented` is a follow-up.

The 4-way independent-Optional split (weight-only / bias-only / both / neither) is intentionally **out of scope** here. PyTorch's `torch.nn.GroupNorm` and `torch.nn.InstanceNorm*` toggle weight and bias together via the `affine` flag, so the canonical user-facing variants are exactly two: affine and no-affine. If a future op surfaces independent weight/bias optionality from PyTorch, that op gets its own 4-way split issue.

This is a manifest-only PR per [trust-model.md](docs/design/trust-model.md): signature changes require human review, so no `tileops/ops/` code lands in this PR.

## Test plan

- [x] AC-VARIANT — Two new variant entries (`GroupNormFwdOpNoAffine`, `InstanceNormFwdOpNoAffine`) added to `tileops/manifest/normalization.yaml` with `variant_of` pointing to their primary entries.

Verification (self-executed by reviewer):

- `pytest tests/test_validate_manifest.py` → 199 passed.
- `python scripts/validate_manifest.py` → All manifest checks passed.
- Structural diff vs `upstream/testbed`: only `tileops/manifest/normalization.yaml` changed; primaries byte-stable; each variant's `variant_of` points directly to its primary (single-level, no chaining); variants share `source.kernel` / `source.op` with primaries.

### CODE REVIEW

File: tileops/manifest/normalization.yaml [hunk 1]
```diff
@@ -454,6 +454,57 @@ GroupNormFwdOp:
     test: tests/ops/test_group_norm.py
     bench: benchmarks/ops/bench_group_norm.py
 
+GroupNormFwdOpNoAffine:
+  ref_api: "torch.nn.functional.group_norm"
+  # No-affine variant: torch.nn.functional.group_norm(x, num_groups, weight=None,
+  # bias=None, eps), matching torch.nn.GroupNorm(affine=False). Split from
+  # the affine primary per the "No Optional[Tensor]" rule (R16,
+  # .claude/domain-rules/manifest-spec.md). torch.nn.GroupNorm toggles
+  # weight and bias together via the affine flag.
```
[the remark this answers is not in the document]
Comment (lcy-seso) [reply tileops/manifest/normalization.yaml:463]: Acknowledged and out of scope for this PR. PR body explicitly documents that the 4-way independent-Optional split (weight-only / bias-only) is intentionally deferred — `nn.GroupNorm`/`nn.InstanceNorm` modules toggle weight+bias together via the affine flag, so the canonical 2-way split (affine=True / affine=False) covers the primary use cases. The Python op layer's affine-identity cache makes weight-only / bias-only paths work in practice via identity substitution; if the manifest needs to formally model those rare partial cases later, a follow-up issue can add the additional 2 variants per op.

File: tileops/manifest/normalization.yaml [hunk 2]
```diff
@@ -500,3 +551,52 @@ InstanceNormFwdOp:
     op: tileops/ops/norm/instance_norm.py
     test: tests/ops/test_instance_norm.py
     bench: benchmarks/ops/bench_instance_norm.py
+
+InstanceNormFwdOpNoAffine:
+  ref_api: "torch.nn.functional.instance_norm"
+  # No-affine variant: torch.nn.functional.instance_norm(x, weight=None,
+  # bias=None, ...), matching torch.nn.InstanceNorm*(affine=False) — the
+  # default for InstanceNorm. Split from the affine primary per the
+  # "No Optional[Tensor]" rule (R16, .claude/domain-rules/manifest-spec.md).
```
[the remark this answers is not in the document]
Comment (lcy-seso) [reply tileops/manifest/normalization.yaml:560]: Acknowledged and out of scope for this PR. PR body explicitly documents that the 4-way independent-Optional split (weight-only / bias-only) is intentionally deferred — `nn.GroupNorm`/`nn.InstanceNorm` modules toggle weight+bias together via the affine flag, so the canonical 2-way split (affine=True / affine=False) covers the primary use cases. The Python op layer's affine-identity cache makes weight-only / bias-only paths work in practice via identity substitution; if the manifest needs to formally model those rare partial cases later, a follow-up issue can add the additional 2 variants per op.

File: tileops/manifest/normalization.yaml [hunk 3]
```diff
@@ -500,3 +551,52 @@ InstanceNormFwdOp:
...
+  variant_of: InstanceNormFwdOp
+
+  signature:
+    inputs:
+      x: {dtype: "float32 | float16 | bfloat16"}
+    outputs:
+      output: {dtype: "same_as(x)"}
+    params:
+      use_input_stats: {type: bool, default: true}
+      momentum: {type: float, default: 0.1}
```
[the remark this answers is not in the document]
Comment (lcy-seso) [reply tileops/manifest/normalization.yaml:572]: Pre-existing drift, not introduced by this PR. The primary `InstanceNormFwdOp` entry already has `use_input_stats`/`momentum` declared in its params block (look at the testbed-side manifest). Since the variant inherits primary's param shape for spec consistency, dropping them only on the variant would create asymmetry. Whether the primary should drop these stale params (their Python implementation was removed in PR #1256's normalization cleanup) is a separate manifest-spec question — out of scope for this variant_of split. If you want to track that as a follow-up: file a manifest-only PR to align the primary with the current Python signature.

**Review by Ibuki-wind — APPROVED** [review]
Nightshift autonomous review: HEAD ed005ea, CI green, 0 unresolved threads (3 bot threads triaged: 2 partial-Optional concerns out-of-scope per PR body, 1 stale-params concern is pre-existing primary drift). Manifest-only variant_of split is R16-compliant. Approving for squash-merge into testbed.

**Review by Ibuki-wind — APPROVED** [review]
Nightshift autonomous review: HEAD ed005ea, CI green, 0 unresolved threads. Approving for squash-merge into testbed.
